packages/fxa-auth-server/test/local/senders/mjml-emails.ts

```diff
@@ -0,0 +1,994 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const ROOT_DIR = '../../..';
+
+import { assert } from 'chai';
+import mocks from '../../mocks';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+import { URL } from 'url';
+
+const config = require(`${ROOT_DIR}/config`).getProperties();
+if (!config.smtp.prependVerificationSubdomain.enabled) {
+  config.smtp.prependVerificationSubdomain.enabled = true;
+}
+if (!config.smtp.sesConfigurationSet) {
+  config.smtp.sesConfigurationSet = 'ses-config';
+}
+config.smtp.subscriptionTermsUrl = 'http://example.com/terms';
+
+// Force enable the subscription transactional emails
+config.subscriptions.transactionalEmails.enabled = true;
+
+const TEMPLATE_VERSIONS = require(`${ROOT_DIR}/lib/senders/templates/_versions.json`);
+
+const SUBSCRIPTION_TERMS_URL = 'https://example.com/subscription-product/terms';
```

Comment:
I'm not sure we should add all of this stuff until we get to the Subscriptions.

Can we maybe strip out anything that's not cadReminderFirst specific and add each piece in as needed with each template conversion? That'll help clean up this file along the way and make reviewing it a bit easier. Maybe we can have this file living in a gist so we can reference any Typescript wins as other templates are being converted.